module_1/first-react-app/src/components/MovieDetails.js

```diff
@@ -0,0 +1,59 @@
+import React, {useState} from 'react';
+import StarRating from './StarRating';
+import '../css/MovieDetails.css'
+import { NOT_APPLICABLE, UNKNOWN_MOVIE } from '../literals';
+
+const MovieDetails = ({ imageUrl, movieName, releaseYear, rating, duration, description, genres, onClose }) => {
+
+    const trimmedDescription = description.length <= 400 ? description : description.slice(0, 400).trim() + '.......';
```

Comment:
Why do we need `.trim()` here?

Reply:
corrected in commit: d4871b3879a408f1fcc8f54e9322ded924683aaf